tests/optimization/src/TestJs.hx

```diff
@@ -172,12 +172,12 @@ class TestJs {
 		try throw false catch (e:Dynamic) {}
 	}
 
-	@:js('try {throw new js__$Boot_HaxeError(false);} catch( e ) {if (e instanceof js__$Boot_HaxeError) e = e.val;TestJs.use(e);}')
+	@:js('try {throw new js__$Boot_HaxeError(false);} catch( e ) {var e1 = (e instanceof js__$Boot_HaxeError) ? e.val : e;var e2 = e1;TestJs.use(e2);}')
```

Comment:
This `var e2 = e1;` is unfortunate, but I think that's something the analyzer should handle.

Reply:
It probably considers the variable to be a user-var. Try adding `Meta.CompilerGenerated` to it.

Reply:
Will try, thanks!

Reply:
That helped!